Framework/Mobile/LibTest/AppDelegate.m

```diff
@@ -1,41 +0,0 @@
-#import "AppDelegate.h"
-#import "ViewController.h"
-
-#import <CocoaLumberjack/CocoaLumberjack.h>
-#import "Formatter.h"
```

Comment:
I think demos can achieve the same goal outside the project, plus moving it out keeps the project cleaner once you only need the framework.

Also having some non-CocoaPods demos would be something good. Not sure if they can use Carthage, if so that could be the extra Travis build step.